php/php.phing/src/org/netbeans/modules/php/phing/exec/PhingExecutable.java

```diff
@@ -163,10 +161,7 @@ private PhpExecutable getExecutable(String title) {
     }
 
     private List<String> getRunParams(String[] args) {
-        List<String> params = new ArrayList<>(args.length + 2);
-        params.add(LOGGER_PARAM);
-        params.add(LOGGER_COLOR_VALUE);
-        params.addAll(Arrays.asList(args));
+        List<String> params = Collections.unmodifiableList(Arrays.asList(args));
```

Comment:
this changes the semantics of getRunParams - before the change the returned list was modifiable, now its not anymore.

Reply:
I changed it to normal list.